File: content-solutions/graph-representation/graph-friend-list.java
import java.util.*;

class Solution {
    public List<List<Integer>> solution(int n, int[][] pairs) {
        List<List<Integer>> graph = new ArrayList<>();
        for (int i = 0; i < n; i++) graph.add(new ArrayList<>());
        for (int[] e : pairs) {
            graph.get(e[0]).add(e[1]);
            graph.get(e[1]).add(e[0]);
        }
        for (List<Integer> friends : graph) Collections.sort(friends);
        return graph;
    }
}

File: content-solutions/graph-representation/graph-friend-list.js
function solution(n, pairs) {
  const graph = Array.from({ length: n }, () => []);
  for (const [a, b] of pairs) {
    graph[a].push(b);
    graph[b].push(a);
  }
  for (const friends of graph) friends.sort((p, q) => p - q);
  return graph;
}
